src/components/views/dialogs/ResetIdentityDialog.tsx

```diff
@@ -40,23 +31,17 @@ interface ResetIdentityDialogProps {
 /**
  * The dialog for resetting the identity of the current user.
  */
-export function ResetIdentityDialog({
-    onFinished,
-    onReset,
-    onCancelled,
-    variant,
-}: ResetIdentityDialogProps): JSX.Element {
+export function ResetIdentityDialog({ onFinished, onReset, variant }: ResetIdentityDialogProps): JSX.Element {
     const matrixClient = MatrixClientPeg.safeGet();
 
     // Wrappers for ResetIdentityBody's callbacks so that onFinished gets called
     // whenever the reset is done, whether by completing successfully, or by
     // being cancelled
-    const onResetWrapper: MouseEventHandler<HTMLButtonElement> = (...args) => {
-        onReset?.();
+    const onResetWrapper: () => void = () => {
+        onReset();
         onFinished?.();
     };
     const onCancelledWrapper: () => void = () => {
-        onCancelled?.();
         onFinished?.();
     };
```

Comment:
AFAICT `onFinished` is also unused. Can we get rid of it too, and hence these wrappers?

Reply:
Calling onFinished actually closes the dialog(!) so we need it. (And it is part of the interface we must satisfy to be passed in to `Modal.createDialog`.)